socketcan_bridge/src/socketcan_to_topic_nodelet.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Enway GmbH - All Rights reserved.
+ * Proprietary & confidential.
+ */
+
+#include "socketcan_bridge/socketcan_to_topic_nodelet.h"
+
+#include <pluginlib/class_list_macros.h>
+
+PLUGINLIB_EXPORT_CLASS(socketcan_bridge::SocketcanToTopicNodelet, nodelet::Nodelet)
+
+socketcan_bridge::SocketcanToTopicNodelet::~SocketcanToTopicNodelet()
+{
+  driver_->shutdown();
+  driver_.reset();
+}
+
+void socketcan_bridge::SocketcanToTopicNodelet::onInit()
+{
+  ros::NodeHandle nh = getPrivateNodeHandle();
+  std::string can_device;
+  nh.param<std::string>("can_device", can_device, "can0");
+
+  driver_ = boost::make_shared<can::ThreadedSocketCANInterface> ();
```

Comment:
There shouldn't be a space before the parentheses

Reply:
fixed